change/@fluentui-web-components-9c72f362-06d8-4d36-a1db-2f295435878a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Adding react-checkbox to react-examples' preview.js.",
+  "packageName": "@fluentui/web-components",
+  "email": "esteban.230@hotmail.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
This should be a none change - and there likely shouldn't be any code from the web-components with this change from what I can tell. If the prettier change is *expected*, please ensure that the change file reflects the change and that it's "none" so it does not rev the package.